src/le.py

```diff
@@ -3177,10 +3372,17 @@ def start_followers(default_transport):
             if not entry_formatter:
                 entry_formatter = formats.get_formatter('syslog', config.hostname, log_name, log_token)
 
-            # Instantiate the follower
-            follower = Follower(log_filename, entry_filter, entry_formatter, entry_identifier, transport)
-            followers.append(follower)
-    return (followers, transports)
+            # Instantiate the follow_multilog for 'multilog' filename, otherwise the individual follower
+            if multilog_filename:
+                # todo log.info("Multilog log_filename is %s", log_filename)
+                follow_multilog = FollowMultilog(log_filename, entry_filter, entry_formatter, entry_identifier, transport)
+                follow_multilogs.append(follow_multilog)
+                multilog_filename=False
```

Comment:
yes, but I should move this to the top of the loop - I had declared the variable at the top of the function, outside the loop -  will change this.